File: app/src/main/java/cn/ifafu/ifafu/util/ToastUtils.java
package cn.ifafu.ifafu.util;

import android.content.Context;
import android.widget.Toast;

import cn.ifafu.ifafu.base.BaseApplication;

/**
 * create by woolsen on 19/7/12
 */
public class ToastUtils {

    public static void showToastLong(String msg) {
        showToast(BaseApplication.getAppContext(), msg, Toast.LENGTH_LONG);
    }

    public static void showToastLong(int msg) {
        showToast(BaseApplication.getAppContext(), "" + msg, Toast.LENGTH_LONG);
    }

    public static void showToastShort(String msg) {
        showToast(BaseApplication.getAppContext(), msg, Toast.LENGTH_SHORT);
    }

    public static void showToastShort(int msg) {
        showToast(BaseApplication.getAppContext(), "" + msg, Toast.LENGTH_SHORT);
    }

    public static void showToast(Context context, String msg, int duration) {
        Toast.makeText(context, msg, duration).show();
    }
}

File: easymvvm/src/main/java/cn/woolsen/easymvvm/util/ToastUtils.kt
package cn.woolsen.easymvvm.util

import android.content.Context
import android.widget.Toast
import androidx.annotation.StringRes

/**
 * create by woolsen on 19/7/12
 */
object ToastUtils {
    fun showToastLong(context: Context, text: CharSequence) {
        Toast.makeText(context, text, Toast.LENGTH_LONG).show()
    }

    fun showToastLong(context: Context, @StringRes resId: Int) {
        Toast.makeText(context, resId, Toast.LENGTH_LONG).show()
    }

    fun showToastShort(context: Context, text: CharSequence) {
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show()
    }

    fun showToastShort(context: Context, @StringRes resId: Int) {
        Toast.makeText(context, resId, Toast.LENGTH_SHORT).show()
    }
}